apps/accounts/forms/auth.py

```diff
@@ -436,7 +463,10 @@ def clean(self):
                 errors = self._errors[field_name]
                 error_msg = errors[0]
                 if error_msg == 'This field is required.':
-                    error_msg = "Please check the box indicating that you agree with %s's Privacy Policy and Terms of Service." % htk_setting('HTK_SITE_NAME')
+                    error_msg = (
```

Comment:
let's keep what we had before and add `# noqa`

Reply:
Done